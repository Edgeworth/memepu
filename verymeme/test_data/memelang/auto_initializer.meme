struct Empty {}
fn main() i8 {
  var e: auto = Empty{};
}
