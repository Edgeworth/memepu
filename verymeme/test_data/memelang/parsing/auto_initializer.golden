File
 Fn
  FnSig
   Typename(main)
   Type(i8)
  StmtBlk
   VarDefn
    VarDecl
     VarRef(e)
     Type()
    Op(FN_CALL)
     Type(Empty)
     FnCallArgs
      CompoundLit
 Struct
  Typename(Empty)
